bookkeeper-server/src/test/java/org/apache/bookkeeper/bookie/BookieJournalForceTest.java

```diff
@@ -320,6 +322,7 @@ public void testForceLedger() throws Exception {
         Bookie.checkDirectoryStructure(Bookie.getCurrentDirectory(journalDir));
 
         ServerConfiguration conf = TestBKConfiguration.newServerConfiguration();
+        conf.setJournalFlushWhenQueueEmpty(false);
```

Comment:
it is still unclear to me how this would cause the flakiness? can you explain more?